entity-framework/core/what-is-new/index.md

```diff
@@ -35,6 +35,46 @@ modelBuilder.Entity<ProductDetails>().ToTable("Products");
 
 ### Owned types
```

Comment:
Let's keep just a super quick overview in the what's new page, and add a link to the new docs page there.